### PR TITLE
Fixes XMLNS declaration in SVG element, removes redundant declaration 

### PR DESCRIPTION
**Pull request by lancekruegger**
The "xlms" declared was a typo. It should be XMLNS or XML namespace. I also noted that this was appearing twice in each SVG element due to it being repeated in common.js also reflected in this PR.

### CODE REVIEW

**Comment by lancekruegger** [comment]
Sorry, noted the other code from master was included. :-(

File: lib/common.js [hunk 1]
```diff
@@ -35,9 +35,6 @@ module.exports = utils = Class.extend({
   // alternate to one level deep
   _make: function (tag, props, data) {
     var el = '<' + tag;
-    if (tag === 'svg') {
```
Comment (alfredkam) [inline lib/common.js]: keep this snippet, and change xlms -> xlmns

File: lib/bubble.js [hunk 2]
```diff
@@ -6,7 +6,7 @@ var bubble = module.exports = ArcBase.extend({
         var svg = this._make('svg',{
             width: chart.width,
             height: chart.height,
-            xlms: 'http://www.w3.org/2000/svg',
+            xmlns: 'http://www.w3.org/2000/svg',
```
Comment (alfredkam) [inline lib/bubble.js]: just remove the xlms / xlmns line, refer to the response below

File: lib/bar.js [hunk 3]
```diff
@@ -32,7 +32,7 @@ var bar = module.exports = Base.extend({
             width: chart.width,
             height: chart.height,
             //No view box?
-            xlms: 'http://www.w3.org/2000/svg',
+            xmlns: 'http://www.w3.org/2000/svg',
```
Comment (alfredkam) [inline lib/bar.js]: just remove the xlms / xlmns line, refer to the response below

File: lib/arcBase.js [hunk 4]
```diff
@@ -33,7 +33,7 @@ var arc = module.exports = Base.extend({
             width: chart.width,
             height: chart.height,
             //No view box?
-            xlms: 'http://www.w3.org/2000/svg',
+            xmlns: 'http://www.w3.org/2000/svg',
```
Comment (alfredkam) [inline lib/arcBase.js]: just remove the xlms / xlmns line, refer to the response below

File: lib/sparkLine.js [hunk 5]
```diff
@@ -55,7 +55,7 @@ var spark = module.exports = Base.extend({
         svg = this._make('svg',{
             width: opts.chart.width,
             height: opts.chart.height,
-            xlms: 'http://www.w3.org/2000/svg',
+            xmlns: 'http://www.w3.org/2000/svg',
```
Comment (alfredkam) [inline lib/sparkLine.js]: just remove the xlms / xlmns line, refer to the response above

**Comment by alfredkam** [comment]
just make that quick change for me then i can merge it, thx @lancekruegger !

**Comment by lancekruegger** [comment]
Okay, thanks for the comments. I have gone ahead and addressed them.

**Comment by hasalfred** [comment]
once you've sync with master i can merge it

**Pull request merged**